
INFO:root:Epoch 1
/home/brett/anaconda3/lib/python3.8/site-packages/torch/utils/data/dataloader.py:558: UserWarning: This DataLoader will create 10 worker processes in total. Our suggested max number of worker in current system is 4, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(_create_warning_msg(
Process Process-1:
Traceback (most recent call last):
  File "/home/brett/anaconda3/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/brett/anaconda3/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/brett/Desktop/tutorials/ijepa/ijepa/main.py", line 75, in process_main
    app_main(args=params)
  File "/home/brett/Desktop/tutorials/ijepa/ijepa/src/supervised_train.py", line 406, in main
    save_checkpoint(epoch + 1)
  File "/home/brett/Desktop/tutorials/ijepa/ijepa/src/supervised_train.py", line 267, in save_checkpoint
    'model': model.state_dict(),
NameError: free variable 'model' referenced before assignment in enclosing scope
INFO:root:Validation Loss: 1.680, Validation Accuracy: 41.48%